src/edu/umass/cs/aws/support/Route53.java

```diff
@@ -62,13 +62,13 @@ public static void main(String[] args) throws Exception {
     listRecordSetsForHostedZone();
   }
 
-  private void test() {
-    route53.createHostedZone(new CreateHostedZoneRequest()
-            .withName("myDomainName.com")
-            .withCallerReference(CALLER_REFERENCE)
-            .withHostedZoneConfig(new HostedZoneConfig()
-            .withComment("my first Route 53 hosted zone!")));
-  }
+//  private void test() {
```

Comment:
The whole file is full of unused code, but that one was generating a warning so I commented it out. Next time I'll probably just suppress the warning.